.github/workflows/issue_to_project.yml

```diff
@@ -0,0 +1,16 @@
+name: Add issues to project
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.6.1
+        with:
+          project-url: https://github.com/orgs/CQCL-DEV/projects/19
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
You will need to add a token as a github actions secret on this repo with this name. See https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic (currently only a "classic" token will work, and it needs to have "repo" and "project" permissions.